canonical-web-and-design/vanilla-framework
# Checkbox alignment classes

## Done

- Provide is-* classes that align checkboxes / radios to any size of heading / text.
(This is just to look at the percy diff, not ready for review yet).

Fix alignment of text that is not wrapped by any tag and is thus not aligned to the baseline grid (as in - tables etc.)

## QA

- Pull code
- Run `./run serve --watch`
- Open /base/forms/
- QA examples; review class naming and verify you're happy with it

## Details

[List of links to issues/bugs and cards if needed]

## Screenshots

![image](https://user-images.githubusercontent.com/2741678/64253303-ea608380-cf14-11e9-9405-536df5eb9fbb.png)

![image](https://user-images.githubusercontent.com/2741678/64253329-fb10f980-cf14-11e9-86bd-dc3c6f257c60.png)
